platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/SignedStateFileReadWriteTest.java

```diff
@@ -160,100 +156,6 @@ void writeSavedStateToDiskTest() throws IOException {
         assertTrue(exists(addressBookFile), "address book file should exist");
     }
 
-    @Test
-    @DisplayName("getSignedStateBaseDirectory() Test")
-    void getSignedStateBaseDirectoryTest() {
-        changeConfigAndConfigHolder("data/saved");
-
-        assertEquals(getAbsolutePath("./data/saved"), getSignedStatesBaseDirectory(), "unexpected saved state file");
-
-        changeConfigAndConfigHolder("foo/bar/baz");
-
-        assertEquals(getAbsolutePath("./foo/bar/baz"), getSignedStatesBaseDirectory(), "unexpected saved state file");
-
-        changeConfigAndConfigHolder("data/saved");
-    }
-
-    @Test
-    @DisplayName("getSignedStatesDirectoryForApp() Test")
-    void getSignedStatesDirectoryForAppTest() {
```

Comment:
Why do you remove all that tests?

Reply:
These tests tested methods that were only called by these tests. I removed both the platform code and the tests.